lib/hammer_cli_csv/content_hosts.rb

```diff
@@ -595,6 +596,7 @@ def columns_to_csv(csv)
       def hypervisor_from_line(line)
         hypervisor = line[HOST] if !line[HOST].nil? && !line[HOST].empty?
         hypervisor ||= line[GUESTOF] if !line[GUESTOF].nil? && !line[GUESTOF].empty?
+        hypervisor = namify(hypervisor, 1) if !hypervisor.nil? && !hypervisor.empty?
```

Comment:
No argument against. I will adjust the pattern in future PRs, if that is preferred.